.gitignore

```diff
@@ -118,3 +118,6 @@ docbuild/
 
 # OSX directory info files
 .DS_Store
+
+# plot_quickstart.py
+filename.png
```

Comment:
nit: I believe it gets generated under `examples` so perhaps `**/filename.png` helps to account for such files.